analyzers/src/SonarAnalyzer.Common/Helpers/ParameterLoader.cs

```diff
@@ -39,84 +36,32 @@ internal static class ParameterLoader
          * - diffing the contents of the configuration file
          * - associating the file with a unique identifier for the build project
          */
-        internal static void SetParameterValues(ParametrizedDiagnosticAnalyzer parameteredAnalyzer,
-            AnalyzerOptions options)
+        internal static void SetParameterValues(ParametrizedDiagnosticAnalyzer parameteredAnalyzer, SonarLintXmlReader sonarLintReader)
```

Comment:
This is not a reader of SonarLint (extension)
```suggestion
        internal static void SetParameterValues(ParametrizedDiagnosticAnalyzer parameteredAnalyzer, SonarLintXmlReader sonarLintXml)
```